utils/python/CIME/XML/entry_id.py

```diff
@@ -71,6 +71,24 @@ def set_value(self, vid, value, subgroup=None):
         else:
             return None
 
+    def _set_value(self, node, vid, value, subgroup=None):
```

Comment:
Looks like we have duplicates of these set_value methods. I'm going to assume the news ones are the right versions.